src/components/EditGrower.js

```diff
@@ -161,12 +161,22 @@ const EditGrower = (props) => {
               ))}
             </Grid>
           ))}
+          <TextField
```

Comment:
This is ok, but it's best to extend the `inputs` array defined above. Using that ensures all the inputs are handled in the same way and reduces code duplication.
You will need to add an optional `multiline` flag to objects in `inputs`.